gradle/libs.versions.toml

```diff
@@ -33,7 +33,6 @@ auto-service-compiler = { module = "dev.zacsweers.autoservice:auto-service-ksp",
 auto-service-annotations = { module = "com.google.auto.service:auto-service-annotations", version = "1.1.1" }
 binary-compatibility-validator-gradle-plugin = { module = "org.jetbrains.kotlinx.binary-compatibility-validator:org.jetbrains.kotlinx.binary-compatibility-validator.gradle.plugin", version = "0.13.2" }
 cklib-gradle-plugin = { module = "co.touchlab:cklib-gradle-plugin", version = "0.3.0" }
-coil-compose = { module = "io.coil-kt:coil-compose", version = "2.5.0" }
```

Comment:
Noticed this is unused (unfortunately!)

Reply:
Ah from emoji probably, before it moved to Redwood.